homeassistant/components/sharkiq/strings.json

```diff
@@ -20,7 +20,6 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
-      "already_configured_account": "[%key:common::config_flow::abort::already_configured_account%]",
```

Comment:
The flow uses the helper `_abort_if_unique_id_configured`. Please rename the key to `already_configured`.

Reply:
Did not know that.
Thanks.